tasks/bzr.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Set etckeeper bzr whoami
+  become: yes
+  command: "etckeeper vcs whoami '{{etckeeper_vcs_username}} <{{etckeeper_vcs_email}}>'"
+  tags: [ 'etckeeper_creds' ]
```

Comment:
Tags currently don’t work in include files. http://www.ansible.com/blog/ansible-2.0-launch Also the DebOps project uses a specific format for tags. I would suggest `role::etckeeper:commit_options`.